gatorgrade/output/output.py

```diff
@@ -208,6 +208,51 @@ def calculate_deadline_time_dif(older_time: datetime, latest_time: datetime):
 
 
 def run_checks(checks: List[Union[ShellCheck, GatorGraderCheck]], deadline) -> bool:
```

Comment:
Looks like the function signature here might be missing a few of the needed parameters, as well as the `Args` part of the docstring.